install.sh

```diff
@@ -0,0 +1,231 @@
+#      and/or other materials provided with the distribution.
+#   3. Neither the name of Shadow Robot Company Ltd nor the names of its contributors
+#      may be used to endorse or promote products derived from this software without
+#      specific prior written permission.
+# This software is provided by Shadow Robot Company Ltd "as is" and any express
+# or implied warranties, including, but not limited to, the implied warranties of
+# merchantability and fitness for a particular purpose are disclaimed. In no event
+# shall the copyright holder be liable for any direct, indirect, incidental, special,
+# exemplary, or consequential damages (including, but not limited to, procurement of
+# substitute goods or services; loss of use, data, or profits; or business interruption)
+# however caused and on any theory of liability, whether in contract, strict liability,
+# or tort (including negligence or otherwise) arising in any way out of the use of this
+# software, even if advised of the possibility of such damage.
+
+set -e
+
+# Colors for terminal output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+YELLOW='\033[1;33m'
+NC='\033[0m' 
+
+DEVELOPMENT=false
+KERNEL_URL="https://s3.eu-west-2.amazonaws.com/com.shadowrobot.eu-west-2.public/linux-image-6.5.2-rt8_6.5.2-3_amd64.deb"
+KERNEL_LOCATION="/tmp/rtkernel.deb"
+
+
+# Print colour functions
+print_red() {
+    echo -e "${RED}$1${NC}"
+}
+
+print_green() {
+    echo -e "${GREEN}$1${NC}"
+}
+
+print_yellow() {
+    echo -e "${YELLOW}$1${NC}"
+}
+
+print_blue() {
+    echo -e "${BLUE}$1${NC}"
+}
+
+print_startup_message() {
+    print_yellow "================================================================="
+    print_yellow "|                                                               |"
+    print_yellow "|                 Shadow Dexee Deployment Tool                  |"
+    print_yellow "|                                                               |"
+    print_yellow "================================================================="
+    print_yellow ""
+    print_yellow "Possible options:"
+    print_yellow "  * --dev           Install the development container"
+    print_yellow "  * --help          Print this help"
+}
+
+install_apps() {
+    print_yellow "Updating package repositories..."
+    sudo apt update >/dev/null || { print_red "Failed to update repositories"; exit 1; }
+    
+    print_yellow "Upgrading system packages..."
+    sudo apt upgrade -y >/dev/null || { print_red "Failed to upgrade packages"; exit 1; }
+    
+    print_yellow "Installing required packages (Terminator, Docker, OpenSSH, Curl)..."
+    sudo apt install -y terminator docker.io openssh-server curl || { print_red "Failed to install packages"; exit 1; }
+    
+    print_green "Package installation complete"
+}
+
+install_kernel() {
+    print_yellow "Installing RT kernel..."  
+    # Check if kernel is already installed
+    if dpkg -l | grep -q "linux-image-6.5.2-rt8"; then
```

Comment:
Should we make this string a variable that is also used to define the 
`KERNEL_URL="https://s3.eu-west-2.amazonaws.com/com.shadowrobot.eu-west-2.public/linux-image-6.5.2-rt8_6.5.2-3_amd64.deb"`?